docs/spec/index.md

```diff
@@ -1,54 +1,105 @@
-| location_sequence | Non-negative Integer | Required | Order of locations, including both operational locations and stops, for a particular deadhead. The values must increase along the trip but do not need to be consecutive. |
-| shape_dist_traveled | Non-negative Float | Optional | Actual distance traveled along the associated shape, from the first location to the location specified in this record. |
+### Structure
+
+The fields in all Supplement files match those defined in the corresponding file's [GTFS specification](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md).
+
+The overall premise is to update and add data to GTFS, which is accomplished by either updating matching values or adding rows entirely.
+
+Each entry in a Supplement file is paired with its matching entry in the corresponding GTFS file using the [GTFS file's Primary Key](https://gtfs.org/schedule/reference/#dataset-attributes), i.e. the fields that uniquely identify the row. If no match is found, the Supplement file's entry is added to the GTFS file in its entirety.
+
+The standard also supports the removal of rows in their entirety.
+
+### Evaluation
+
+Each row in a Supplement file shall be evaluated as follows:
+
+1. If the row's Primary Key is defined in the corresponding GTFS file, and the `TODS_delete` field is defined and equal to `1`, remove the corresponding row from the GTFS file.
+2. If the row's Primary Key is defined in the corresponding GTFS file, and the `TODS_delete` field is not defined or not equal to `1`, set or update any fields with defined values in the Supplement file to those values in the corresponding GTFS file.
+3. If the row's Primary Key is NOT defined in the corresponding GTFS file, add the row to the corresponding GTFS file.
+
+In other words, where a Primary Key matches, the row is either removed or any non-empty values in the row are used to _update_ the corresponding GTFS values. Where a Primary Key match does not exist, the entire row is added.
+
+### Example
+
+GTFS `stops.txt`:
+
+```csv
+stop_id,stop_name,stop_desc,stop_url
+1,One,Unmodified in TODS,example.com/1
+2,Two,Deleted in TODS,example.com/2
+3,Three,Will be modified in TODS,example.com/3
+```
+
+TODS `stops_supplement.txt`:
+
+```CSV
+stop_id,stop_name,stop_desc,TODS_delete
+2,,,1
+3,,Has been modified by TODS,
+4,Four,New in TODS,
+```
+
+Effective `stops.txt` after merging the supplement file:
+
+```CSV
+stop_id,stop_name,stop_desc,stop_url
+1,One,Unmodified in TODS,example.com/1
+3,Three,Has been modified by TODS,example.com/3
+4,Four,New in TODS,
+```
+
+_Note that the station name "Three" was not modified, and the whole column stop_url was omitted and not modified._
+
+### Implications and Guidance
+
+- As blank fields are ignored, data to be removed should either be overwritten with a new value or have their entire row deleted using the `TODS_delete` field.
```

Comment:
See [https://github.com/cal-itp/operational-data-standard/pull/61#discussion_r1513587956](https://github.com/cal-itp/operational-data-standard/pull/61#discussion_r1513587956).

This is not currently possible in the proposed version of the spec, but since a consumer could simply ignore this data, we struggled to find a use case that warranted immediate inclusion.

I think @skyqrose's proposal of a `tods_blank` field is the most promising, but has some implementation details worth reviewing before including.

The consensus was to move forward and track this as an open item should it present itself as an item needing to be addressed in a subsequent update.

Reply:
Thanks! That discussion addresses my concerns.